client/components/EventsListContainer/index.jsx

```diff
@@ -75,7 +75,8 @@ class EventsListComponent extends React.Component {
                                 loadMoreEvents={this.props.loadMoreEvents}
                                 selectedEvents={this.props.selectedEvents}
                                 onEventSelectChange={this.props.onEventSelectChange}
-                                privileges={privileges} />
+                                privileges={privileges}
+                                session={session} />
```

Comment:
Thanks to react-virtualized we only render events that we can display.
And in a way, there are already connected to the store, through `EventListContainer`.